core/src/impl/Kokkos_Profiling_Interface.hpp

```diff
@@ -68,6 +68,80 @@ struct SpaceHandle {
   char name[64];
 };
 
+} // end namespace Profiling
+
+namespace Tuning {
+struct ValueSet {
```

Comment:
@jeffmiles63 , do we have any rules for this from that "best practices" meeting? I'm thinking something like

`KokkosCType_ValueSet`

So that we can identify why this type isn't being namespace-qualified.